### PR TITLE
docs - update sql ref pages for ALTER/CREATE/DROP FDW, FOREIGN TABLE; IMPORT FOREIGN SCHEMA

### PR DESCRIPTION
**Pull request by lisakowen**
this PR updates the listed sql command reference pages to align with the postgres v12 ref page content.  greenplum-specific content includes the mpp_execute option and statement that gporca does not support foreign tables.

questions/notes
- gporca does not currently support foreign tables, but should by greenplum 7 GA.  will update the docs when gporca support lands/merged.
- do we support CREATE FOREIGN TABLE with the following clause?  it is not in the file doc/src/sgml/ref/create_foreign_table.sgml.
    ```
    IN ( <partition_bound_expr> [, ...] ) |
    FROM ( { <partition_bound_expr> | MINVALUE | MAXVALUE } [, ...] )
      TO ( { <partition_bound_expr> | MINVALUE | MAXVALUE } [, ...] ) |
    WITH ( MODULUS <numeric_literal>, REMAINDER <numeric_literal> )
    ```

### CODE REVIEW

**Comment by lisakowen** [comment]
doc review site links (behind VPN), can navigate to others from there:
- alter fdw:  https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/sqlref-fdw/greenplum-database/GUID-ref_guide-sql_commands-ALTER_FOREIGN_DATA_WRAPPER.html
- alter foreign table:  https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/sqlref-fdw/greenplum-database/GUID-ref_guide-sql_commands-ALTER_FOREIGN_TABLE.html
- import foreign schema:  https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/sqlref-fdw/greenplum-database/GUID-ref_guide-sql_commands-IMPORT_FOREIGN_SCHEMA.html

**Comment by shivzone** [comment]
@adam8157 @zxuejing can you review this PR since you both have most recent contet on Foreign tables.

**Comment by adam8157** [comment]
ORCA already supports the foreign tables https://github.com/greenplum-db/gpdb/pull/14287

**Comment by adam8157** [comment]
@huiliang-liu Does the main branch support `IMPORT FOREIGN SCHEMA` and the below syntax?
```
IN ( <partition_bound_expr> [, ...] ) |
FROM ( { <partition_bound_expr> | MINVALUE | MAXVALUE } [, ...] )
  TO ( { <partition_bound_expr> | MINVALUE | MAXVALUE } [, ...] ) |
WITH ( MODULUS <numeric_literal>, REMAINDER <numeric_literal> )
```

Others look good to me.

File: gpdb-doc/markdown/ref_guide/sql_commands/ALTER_FOREIGN_TABLE.html.md [hunk 1]
```diff
@@ -5,36 +5,39 @@ Changes the definition of a foreign table.
...
 
-where action is one of:
-
-```
-
-    ADD [ COLUMN ] <column_name> <column_type> [ COLLATE <collation> ] [ <column_constraint> [ ... ] ]
+    ADD [ COLUMN ] <column_name> <data_type> [ COLLATE <collation> ] [ <column_constraint> [ ... ] ]
     DROP [ COLUMN ] [ IF EXISTS ] <column_name> [ RESTRICT | CASCADE ]
-    ALTER [ COLUMN ] <column_name> [ SET DATA ] TYPE <data_type>
+    ALTER [ COLUMN ] <column_name> [ SET DATA ] TYPE <data_type> [ COLLATE collation ]
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/ALTER_FOREIGN_TABLE.html.md:21]: collation -> &lt;collation>

**Review by dyozie — APPROVED** [review]

**Comment by huiliang-liu** [comment]
> @huiliang-liu Does the main branch support `IMPORT FOREIGN SCHEMA` and the below syntax?
> 
> ```
> IN ( <partition_bound_expr> [, ...] ) |
> FROM ( { <partition_bound_expr> | MINVALUE | MAXVALUE } [, ...] )
>   TO ( { <partition_bound_expr> | MINVALUE | MAXVALUE } [, ...] ) |
> WITH ( MODULUS <numeric_literal>, REMAINDER <numeric_literal> )
> ```
> 
> Others look good to me.

Yes, support `IMPORT FOREIGN SCHEMA` and the partition table related syntax.

**Comment by lisakowen** [comment]
thank you for reviewing @huiliang-liu and @adam8157 .  i removed the statement about GPORCA support.  note that i will submit a different PR for other doc updates related to gporca support for foreign tables when that implementation is complete.

**Pull request merged**